bluetooth-permissions/src/androidMain/kotlin/BluetoothPermissionManager.kt

```diff
@@ -38,25 +38,29 @@ import kotlin.time.Duration
  * @param settings the [Settings] to apply to this manager.
  * @param coroutineScope the [CoroutineScope] of this manager.
  */
-actual class DefaultBluetoothPermissionManager(context: Context, bluetoothAdapter: BluetoothAdapter?, settings: Settings, coroutineScope: CoroutineScope) :
-    BasePermissionManager<BluetoothPermission>(BluetoothPermission, settings, coroutineScope) {
+actual class DefaultBluetoothPermissionManager(
+    context: Context,
+    bluetoothAdapter: BluetoothAdapter?,
+    settings: Settings,
+    val useLocationPermission: Boolean,
```

Comment:
This is inconsistent with other permissions that have settings. The right approach would be to modify the BluetoothPermission object to contain the settings. Take a look at `LocationPermission` for instance